DESCRIPTION

```diff
@@ -10,36 +10,20 @@ Authors@R: c(
     person("Evgeni", "Chasnovski", email = "evgeni.chasnovski@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-1617-4019")),
     person("Will", "Hopper", email = "wjhopper510@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-7848-1946")),
     person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462")),
-    person("Ziyue", "Yang", email = "zyang2k@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-9299-8327")),
```

Comment:
Oops. We don't want to remove all of these people! Can someone please make sure these names get added back in?